### PR TITLE
[CFM-1077] Activity Stream design fixes

### PR DESCRIPTION
**Pull request by guytepper**
https://issuetracker.amplexor.com/jira/browse/CFM-1077

### CODE REVIEW

**Comment by guytepper** [comment]
Changes so far:

**Before:**
image

**After:**
image


**Before:**
image

**After:**
image

**Comment by guytepper** [comment]
@ordavidil Ready for review.

File: project/profiles/capacity4more/modules/c4m/ds/c4m_ds/layouts/c4m_ds_4_8_fluid_stacked/c4m-ds-4-8-fluid-stacked.tpl.php [hunk 1]
```diff
@@ -13,16 +13,16 @@
 
 <div class="content clearfix">
   <<?php print $header_wrapper; ?> class="group-header col-xs-12 <?php print $header_classes; ?>">
-    <?php print $header; ?>
+  <?php print $header; ?>
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/ds/c4m_ds/layouts/c4m_ds_4_8_fluid_stacked/c4m-ds-4-8-fluid-stacked.tpl.php]: please revert the indentation change, thanks.

File: project/profiles/capacity4more/modules/c4m/ds/c4m_ds/layouts/c4m_ds_4_8_fluid_stacked/c4m-ds-4-8-fluid-stacked.tpl.php [hunk 2]
```diff
@@ -13,16 +13,16 @@
...
   </<?php print $header_wrapper; ?>>
   <<?php print $left_wrapper; ?> class="group-left col-xs-4 <?php print $left_classes; ?>">
-    <?php print $left; ?>
+  <?php print $left; ?>
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/ds/c4m_ds/layouts/c4m_ds_4_8_fluid_stacked/c4m-ds-4-8-fluid-stacked.tpl.php]: please revert the indentation change, thanks.

File: project/profiles/capacity4more/modules/c4m/ds/c4m_ds/layouts/c4m_ds_4_8_fluid_stacked/c4m-ds-4-8-fluid-stacked.tpl.php [hunk 3]
```diff
@@ -13,16 +13,16 @@
...
   </<?php print $left_wrapper; ?>>
-  <<?php print $right_wrapper; ?> class="group-right col-xs-8 <?php print $right_classes; ?>">
-    <?php print $right; ?>
+  <<?php print $right_wrapper; ?> class="group-right col-xs-8 message-summary<?php print $right_classes; ?>">
+  <?php print $right; ?>
   </<?php print $right_wrapper; ?>>
   <<?php print $footer_wrapper; ?> class="group-footer col-sm-12 <?php print $footer_classes; ?>">
-    <?php print $footer; ?>
+  <?php print $footer; ?>
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/ds/c4m_ds/layouts/c4m_ds_4_8_fluid_stacked/c4m-ds-4-8-fluid-stacked.tpl.php]: please revert the indentation change, thanks.

File: project/profiles/capacity4more/modules/c4m/ds/c4m_ds/layouts/c4m_ds_4_8_fluid_stacked/c4m-ds-4-8-fluid-stacked.tpl.php [hunk 4]
```diff
@@ -13,16 +13,16 @@
...
+  <?php print $header; ?>
   </<?php print $header_wrapper; ?>>
   <<?php print $left_wrapper; ?> class="group-left col-xs-4 <?php print $left_classes; ?>">
-    <?php print $left; ?>
+  <?php print $left; ?>
   </<?php print $left_wrapper; ?>>
-  <<?php print $right_wrapper; ?> class="group-right col-xs-8 <?php print $right_classes; ?>">
-    <?php print $right; ?>
+  <<?php print $right_wrapper; ?> class="group-right col-xs-8 message-summary<?php print $right_classes; ?>">
+  <?php print $right; ?>
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/ds/c4m_ds/layouts/c4m_ds_4_8_fluid_stacked/c4m-ds-4-8-fluid-stacked.tpl.php]: please revert the indentation change, thanks.

**Comment by guytepper** [comment]
@ordavidil fixed.

File: project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_activity_stream.scss [hunk 5]
```diff
@@ -93,3 +100,7 @@
   @include transform-origin(center center);
   @include rotate(45deg);
 }
+
+.message-summary {
+  padding: 0;
+}
```
Comment (ordavidil) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_activity_stream.scss]: @guytepper 
Please add new line

File: project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_activity_stream.scss [hunk 6]
```diff
@@ -93,3 +100,7 @@
   @include transform-origin(center center);
   @include rotate(45deg);
 }
+
+.message-summary {
```
Comment (ordavidil) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_activity_stream.scss]: @guytepper 
Can this one be inside the `.activity-stream` class ?

**Pull request merged**